frappe/patches/v14_0/event_streaming_deprecation_warning.py

```diff
@@ -0,0 +1,9 @@
+import click
+
+
+def execute():
+	click.secho(
+		"Event Streaming is moved to a separate app in version 15.\n"
```

Comment:
I think it's version 14

Reply:
We'll be releasing this in v15 as it's a major breaking change